ts/data/case.ts

```diff
@@ -629,21 +619,45 @@ function addRequisitionIcon(
   status: QcStatus,
   fragment: DocumentFragment
 ) {
-  const icon = makeIcon(status.icon);
-  addToolTip(icon, makeRequisitionTooltip(requisition));
-  fragment.appendChild(icon);
+  const tooltipIcon = makeTooltip(
+    makeIcon(status.icon),
+    makeRequisitionTooltip(requisition)
+  );
+  fragment.appendChild(tooltipIcon);
 }
 
 function makeRequisitionTooltip(requisition: Requisition) {
-  const toolTipConatiner = new DocumentFragment();
-  toolTipConatiner.appendChild(
+  const tooltipContainer = new DocumentFragment();
+  tooltipContainer.appendChild(
     makeNameDiv(
       requisition.name,
       urls.miso.requisition(requisition.id),
       urls.dimsum.requisition(requisition.id)
     )
   );
-  return toolTipConatiner;
+  return tooltipContainer;
```

Comment:
If you change the type of `contents` in `makeTooltip` to `Node`, it'll accept an `HTMLElement` or `DocumentFragment`, so you can just return `makeNameDiv(...)` here